session/bootstrap.go

```diff
@@ -2181,6 +2199,8 @@ func doDDLWorks(s Session) {
 	mustExecute(s, CreatePlanReplayerTaskTable)
 	// Create stats_meta_table_locked table
 	mustExecute(s, CreateStatsTableLocked)
+	// Create tidb_ttl_table_status table
+	mustExecute(s, CreateTTLTableStatus)
```

Comment:
I think we should also add some codes to create a table when upgrading from an old version.

Reply:
Done